app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @project = Project.find(params[:project_id])
+    @review = Review.new
+  end
+
+  def create
+    @project = Project.find(review_params[:project_id])
+    @review = Review.new(review_params)
+    @review.project = @project
+    if @review.save
+      redirect_to @project
+    else
```

Comment:
Since we don't have a create view... can we just make these local variables?

like project and review instead of @project and @review